addons/survey/views/survey_templates_statistics.xml

```diff
@@ -215,16 +239,36 @@
                     <thead>
                         <tr>
                             <th>User Responses</th>
-                            <th>Occurrence</th>
+                            <!-- todo: decide if we add   t-if="question.question_type != 'datetime' or question.is_scored_question" to th-->
```

Comment:
Did we decide? :smile:

Reply:
We usually try to avoid adding TODOs in the code if we can help it, especially for such a styling concern.
Maybe discuss this with Luc (as I'm not the best person to talk to about styling matters ^^) ?